File: HW/【哈希表】2023Q1A-集五福.py
#
# @hw id=2023Q1A lang=python3
#
# 集五福
#

# @hw code=start
team = input().split(",")
cnt = [0, 0, 0, 0, 0]
for p in team:
    for i, n in enumerate(p):
        cnt[i] += int(n)
print(min(cnt))
# @hw code=end

File: HW/【哈希表】2023Q1A-集五福.java
/*
 * @hw id=2023Q1A lang=java
 *
 * 集五福
 */

// @hw code=start
import java.util.*;

public class Main { 
    public static void main(String[] args) {
        Scanner in = new Scanner(System.in);
        String[] team = in.nextLine().split(",");
        int[] cnt = new int[5];
        for(String p : team) {
            for(int i = 0; i < p.length(); i++) {
                cnt[i] += p.charAt(i) - '0';
            }
        }
        System.out.println(Arrays.stream(cnt).min().getAsInt());
    }
}
// @hw code=end